Readme.md

```diff
@@ -56,7 +56,8 @@ memory_limit=128M
 | AMQP       | `${bref:extra.amqp-php-74}`      | `extension=/opt/bref-extra/amqp.so`        |
 | Blackfire  | `${bref:extra.blackfire-php-74}` | `extension=/opt/bref-extra/blackfire.so`   |
 | Cassandra  | `${bref:extra.cassandra-php-74}` | `extension=/opt/bref-extra/cassandra.so`   |
-| DS         | `${bref:extra.ds-php-74}`        | `extension=/opt/bref-extra/ds.so`          |
+| DS         | `${bref:extra.ds-php-74}`        | `extension=/opt/bref-extra/ds.             |
```

Comment:
There are still changes on that line

Reply:
ho yes, forgot to rebase before push sorry